.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## Pull Request Template for pycountry
+
+Thank you for contributing to pycountry! Before submitting this PR, please ensure you've completed the following steps:
+
+### Pre-Submission Checklist:
+- [ ] Code is properly formatted and adheres to the project's coding guidelines.
+- [ ] Changes have been tested with the latest version of the codebase.
+- [ ] Code changes do not decrease the current test coverage.
+- [ ] All existing tests pass successfully.
+- [ ] New tests (if applicable) are added to cover your changes.
+
+### Local Testing:
+Please run the following commands to ensure your changes do not introduce any new issues:
+- [ ] `pytest` - Runs all the tests to ensure they pass.
+- [ ] `make test` - Executes the custom test suite.
+- [ ] `mypy` - Checks for type consistency.
+- [ ] `pre-commit run --all-files` - Ensures code adheres to pre-commit checks.
+
+### Documentation and Contributors:
+- [ ] Update documentation if your changes impact the API or user experience.
+- [ ] Add your name to the `CONTRIBUTORS.txt` file to acknowledge your contribution to the project.
+
+### Commit Messages and PR:
+- [ ] Use descriptive commit messages that explain your changes.
+- [ ] Reference related issues or PRs if applicable.
+- [ ] Provide a summary of your changes in the PR description.
+
+### Review Request:
+- [ ] Request a review from maintainers or specific contributors.
```

Comment:
Most contributors can't do this, so this section should be removed.

Reply:
Removed!